### ISSUE TITLE
Dashboard for showing hourly usage of machines 

### ISSUE THREAD

**Issue by Xartos** [elastisys/compliantkubernetes-apps#2322]
### Proposed feature

In clusters that are autoscaled it would be great to have a grafana dashboard that shows aggregated usage of machines of a specific type. This to be able to have some indication on how much resources has been used during the last time period.

### Proposed alternatives

A grafana dashboard that shows aggregated usage of machine time of specific sizes

### Definition of done

- [ ] There's a dashboard that shows aggregated usage of machine time of specific sizes

**Comment by pettersv** [comment]
Is it possible to show the accumulated cost of these machines over time?

**Comment by lunkan93** [comment]
I don't think we have a good way of showing costs in a Grafana dashboard, other than hard-coding prices for certain flavors and calculating cost based on that and the usage
